local/bin/py/build/configurations/pull_config.yaml

```diff
@@ -287,7 +287,7 @@
   - repo_name: security-monitoring
     contents:
     - action: security-rules
-      branch: main
+      branch: tanguy.lebarzic/markdown-default-rules
       globs:
         - "security-monitoring/*.json"
```

Comment:
add `*.md` here under globs

Reply:
Thanks @nsollecito, you made me realize the globs were also used to selectively pull from git! Had to make a small change to exclude then from the security_rules processing.